dev_envs/mac-env-setup.md

```diff
@@ -0,0 +1,260 @@
+
+```bash
+sudo xcodebuild -license
+xcode-select --install
+```
+
+### Docker ###
+
+Download the latest stable [Docker for Mac](https://www.docker.com/docker-mac).
+Install it by double-clicking the downloaded `dmg` file and dragging to the
+`Applications` folder.
+
+### iTerm2 (recommended) ###
+
+To set up iTerm2 - a MacOS terminal replacement with some nifty features -
+visit the [iTerm2 download page](https://www.iterm2.com/downloads.html) to
+get the latest stable release. Install it by double-clicking the downloaded
+`dmg` file and dragging to the `Applications` folder.
+
+Once installed, run the iTerm2 app and select the menu option
+`iTerm2 > Make iTerm2 Default Term`.
+
+NOTE: You may also `Make iTerm2 Default Term` on your standard user account.
+
+### VPN configuration ###
+
+Once you're set up with the lab (CAL) VPN connection, we recommend you set up
+split-tunneling. Instructions and a generator script are located in the
+[vpn-config-gen repository](https://github.com/cisagov/vpn-config-gen).
+
+## User environment setup ##
+
+For this section and following sections, log back in as your standard user.
+
+Note: if you installed iTerm2 above, run the iTerm2 app and select the menu
+option `iTerm2 > Make iTerm2 Default Term` to make it your default too.
+
+### Automatic package installation (recommended) ###
+
+To set up a dev environment via the [CISA `laptop` script](https://github.com/cisagov/laptop/),
+execute the following in your terminal:
+
+```sh
+bash <(curl --silent https://raw.githubusercontent.com/cisagov/laptop/master/laptop)
+```
+
+This will `curl` the current install files and script. You can specify
+additional packages to install by creating a `Brewfile.local`, as described in
+the Readme for the `laptop` repo where an example is provided.
+
+### Manual package installation (alternative) ###
+
+Open Terminal and install `brew` per the instructions from
+[Homebrew](https://brew.sh). You will probably have to do this with `sudo`.
+
+```sh
+/bin/bash -c "$(curl --fail --silent --show-error --location https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
+```
+
+After installing Brew, you'll want to install other useful packages. We
+recommend installing all the packages specified in the [CISA `laptop` script repository](https://github.com/cisagov/laptop/).
+
+### Environment configuration ###
+
+For portability between computers and environments, you may want to switch
+from the usual dotfiles setup to using `stow` to be able to use a personal
+repository and then map your dotfiles to that directory. This makes it much
+easier to sync and set up.
+
+Using a `stow`-based setup also allows for some nifty features like
+subdirectories to split out useful files into a more modular approach.
+
+#### Prerequisites ####
+
+These are automatically installed if you used the Automatic installation
+script above.
+
+```sh
+brew install stow
+brew install s3cmd
+brew install figlet
+brew install pinentry-mac
```

Comment:
I'm not sure if this was intentional for clarity, but I believe these can be rolled into one command.
```suggestion
brew install figlet pinentry-mac s3cmd stow
```